pysand/erosion.py

```diff
@@ -113,6 +151,15 @@ def welded_joint(v_m, rho_m, Q_s, D, d_p, h, alpha=60, K=2e-9, n=2.6, rho_t=7850
     :return: E_up: Erosion rate flow facing part of weld [mm/year]
     :return: E_down: Erosion rate downstream of weld [mm/year]
     '''
+
+    # Unique input validation
+    if not isinstance(h, (int, float)):
+        raise exc.FunctionInputFail('{} is not a number'.format(h))
+
```

Comment:
This could be validated as well: 0 < h < D
Ref earlier comment: It could be validated in general function.